### PR TITLE
Add views for kaminari to render pagination in style.

### PR DESCRIPTION
**Pull request by timurvafin**
Views downloaded directly from https://github.com/amatsuda/kaminari_themes using `bin/rails g kaminari:views foundation`

### CODE REVIEW

**Comment by vast** [comment]
@timurvafin, could you please clarify why we need them in our repo? Why they do not follow ruby style guide?

**Comment by timurvafin** [comment]
@vast since we have `kaminari` and `foucndation` in `rails-base` it make sense to make generated pagination view like it should.

Here is pagination without foundation styles:


And here is it with nice styles:

**Comment by vast** [comment]
@timurvafin, ok then. But can we at least fix their code style?

File: app/views/kaminari/_last_page.html.slim [hunk 1]
```diff
@@ -0,0 +1,3 @@
+li
+  = link_to_unless(current_page.last?,
+                   raw(t 'views.pagination.last'), url, { :remote => remote })
```
Comment (vast) [inline app/views/kaminari/_last_page.html.slim]: could omit `{}`

File: app/views/kaminari/_first_page.html.slim [hunk 2]
```diff
@@ -0,0 +1,3 @@
+li
+  = link_to_unless(current_page.first?,
+                   raw(t 'views.pagination.first'), url, :remote => remote)
```
Comment (vast) [inline app/views/kaminari/_first_page.html.slim]: hash syntax

**Comment by vast** [comment]
ok

**Comment by timurvafin** [comment]
@vast change log?)

**Comment by vast** [comment]
@timurvafin, that would be great, but not necessarily. Personally, I don't think kaminari foundation theme is a notable change. But that's definitely is up to a contributor.

Thanks, please :shipit:

**Pull request merged**